docs/blueprints/EP000.rst

```diff
@@ -0,0 +1,153 @@
+
+
+Sections that a blueprint should have
+*************************************
+
+Header
+======
+This section must use the template available in the Blueprint Header section of this blueprint.
+
+Abstract
+========
+A short (~200 word) description of the technical issue being addressed.
+
+Motivation
+==========
+It should clearly explain why the blueprint is being proposed describing any technical issue that is intended to be covered by the proposal.
+
+Rationale
+=========
+The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other environments or scenarios.
+
+Specification
+=============
+The technical specification should describe the aspects of what is being proposed.
+
+Backwards Compatibility
+=======================
+All EPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EP must explain how the author proposes to deal with these incompatibilities.
+
+Security Implications
+=====================
+If there are security concerns in relation to the EP, those concerns should be explicitly written out to make sure reviewers of the EP are aware of them.
+
+Rejected Ideas
+==============
+Throughout the discussion of an EP, various proposed ideas end up not being accepted. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This both helps record the thought process behind the final version of the EP as well as preventing people from bringing up the same rejected ideas again in subsequent discussions.
+ 
+References
+==========
+References -- A collection of URLs used as references through the EP.
+
+Copyright/license
+=================
+
+Types of blueprints
+*******************
+
+Standard: Describes a new feature and its implementation.
+
+Informational: Guidelines or information for the community and developers, but does not propose a new feature.
+
+Epic: Describes a problem and its solution.  
+
+
+################
+Blueprint Header
+################
+
+Every blueprint proposal should use this template to build their own header:
+
+:EP: <\*EP number>
+:Title: <EP title>
+:Authors: <list of authors' names and email addrs>
+:Created: <date created on, in yyyy-mm-dd format>
+:Kytos-Version: <kytos version, in yyyy.v>
+:Status: <Draft | Active | Accepted | Provisional | Deferred | Rejected | Withdrawn | Final | Superseded>
+:Type: <Standards Track | Informational | Process>
+:\**Replaces: <EP number>
+
+\*EP number: should be fixed by Kytos team after the author submit the PR.
+
+\**Optional field.
+
+##########
+References
+##########
+
+[1] https://www.python.org/dev/peps/pep-0001/#pep-audience
+
+#########
+Copyright
+#########
+
+This document is placed in the public domain or under the
+CC0-1.0-Universal license, whichever is more permissive.
```

Comment:
I'm also missing here, some guidance  on "When should we write a blueprint?" Every issue is a blueprint? Every major feature? What is a major feature?

Reply:
I think these points should be in motivation section.